goals: ['goal1']
actual_goal: "goal1"
goal1: {"domain":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/goal1_domain.pddl', 
      "problem":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/prova_problem.pddl',
      "init":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/init_problem.pddl',
      "command":'./ff -p /home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/ -o goal1_domain.pddl -f prova_problem.pddl',
      "folder":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/',
      "plan":"/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/plan.pddl",
      "ontology":"http://www.semanticweb.org/alice/ontologies/2023/10/goal1#",
      "path_onto":"/home/alice/",
      }

goal2: {"domain":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/goal2_domain.pddl', 
      "problem":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/prova_problem.pddl',
      "init":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/init2_problem.pddl',
      "command":'./ff -p /home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/ -o goal2_domain.pddl -f prova_problem.pddl',
      "folder":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/',
      "plan":"/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/plan.pddl",
      "ontology":"http://www.semanticweb.org/alice/ontologies/2023/10/goal2#",
      "path_onto":"/home/alice/",
      }

goal3: {"domain":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/goal3_domain.pddl', 
      "problem":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/prova_problem.pddl',
      "init":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/init3_problem.pddl',
      "command":'./ff -p /home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/ -o goal3_domain.pddl -f prova_problem.pddl',
      "folder":'/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/',
      "plan":"/home/alice/catkin_ws/src/PROPER_Sofar/proper_reasoning/plan.pddl",
      "ontology":"http://www.semanticweb.org/alice/ontologies/2023/10/goal3#",
      "path_onto":"/home/alice/",
      }


      